### DISCUSSION TITLE
Google Satellite as XYZ tiles, partial image load on the app

### DISCUSSION THREAD

**Discussion by RubenSadei** [opengisch/QField#2454] [Q&A]
I tried to load Google Satellite map as XYZ tiles, the image loads, where initially for a second you see the whole map in full screen, after it remains partiale at the bottom of the screen (See screenshot).

![Screenshot_20210622-150707 1](https://user-images.githubusercontent.com/86001956/122931278-055fea00-d36d-11eb-890d-f76b711e6a0b.png)

**Comment by nirvn** [comment]
What's the QField version? Can you provide a test project?

**Reply by RubenSadei** [reply]
The QField Version is "master-dev" (00580a).

**Reply by mbernasocchi** [reply]
can you provide a test project?

**Comment by marioba** [comment]
In order to provide a clearer and more manageable discussion forum and to have all discussions about QField and related products (i.e. QFieldCloud and QFieldSync) in a single place, we transferred all the discussions from the QFieldCloud to the [QField repository](https://github.com/opengisch/QField/discussions).

**Comment by suricactus** [comment]
@RubenSadei marking the discussion as solved due to inactivity.
